### DISCUSSION TITLE
Can't receive email using Postal smtp

### DISCUSSION THREAD

**Discussion by CrypticTide** [postalserver/postal#2400] [Help with using Postal]
just installed postal and i setup all records (dkim txt mx .. ) and i tested email send and all working fine 
but when i tried to send email from my gmail to my postal smtp didn't received anything 
any help ?

**Comment by willpower232** [comment]
You should verify that your port 25 is open from another computer using `telnet postal.yourdomain.com 25`

**Comment by CrypticTide** [comment]
Yes it's open

**Reply by willpower232** [reply]
Did you get a bounce in your gmail or anything?
